src/pages/dar_application/CollaboratorList.js

```diff
@@ -0,0 +1,228 @@
+import { useState, useEffect} from 'react';
+import { a, div, input, label, span, p, i } from 'react-hyperscript-helpers';
+import { v4 as uuidV4} from 'uuid';
+
+export default function CollaboratorList(props) {
+  const {formFieldChange, collaboratorLabel, collaboratorKey, showApproval } = props;
+
+  const [collaborators, setCollaborators] = useState(props.collaborators || []);
+  const [deleteBoolArray, setDeleteBoolArray] = useState(props.deleteBoolArray || []);
+
+  //bootstrap v3 doesn't have outline-button styles, so I'll have to make my own
+  const cancelDeleteButtonStyle = {
+    flex: 1,
+    margin: "2px",
```

Comment:
Optional. As a side note, if you use raw integers here, it defaults to px. This should have the same effect.
```suggestion
    margin: 2,
```
But, we should use single quotes if we do use strings.

Reply:
I'll use single quotes. I prefer to have the units explicitly stated in case a reviewer isn't familiar with the defaults.